cf_remote/packages.py

```diff
@@ -155,8 +156,9 @@ def init_download(self):
                 self.artifacts.append(artifact)
 
     def find(self, tags, extension=None):
-        if not self.extended_data:
+        if self.extended_data is None:
             self.init_download()
+        self.init_download()
```

Comment:
@craigcomstock What is going on with the 2x `init_download()` here?

Reply:
oh dang. bad refactor. passed tests so didn't catch it. thanks!